stable/ignite/values.yaml

```diff
@@ -0,0 +1,50 @@
+# Default values for ignite.
+replicaCount: 2
+
+image:
+  repository: apacheignite/ignite
+  tag: 2.7.0
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+rbac: yes
```

Comment:
Sorry I missed this before, the standards for RBAC are as follows:
https://docs.helm.sh/chart_best_practices/#role-based-access-control

Reply:
I've edited the chart according to the guide. Could you please check it again?